ccd-definition/FixedLists.json

```diff
@@ -55,21 +55,27 @@
   },
   {
     "LiveFrom": "01/01/2017",
-    "ID": "HearingTypeFixedList",
-    "ListElementCode": "Contested interim care order",
-    "ListElement": "Contested interim care order"
+    "ID": "HearingTypeMultiSelectList",
```

Comment:
Please remove `MultiSelect` and `List` from the name. It does not matter how it is rendered, for us it is just `HearingType`. Whether it is displayed as dropdown or radio buttons or anything else is secondary matter.